### ISSUE TITLE
Upgrade graphql module

### ISSUE THREAD

**Issue by langpavel** [calebmer/postgraphql#103]
```
Error: Schema must be an instance of GraphQLSchema. Also ensure that there are not multiple versions of GraphQL installed in your node_modules directory.
    at invariant (/home/langpavel/Projects/work/slackday/breedly/node_modules/graphql/jsutils/invariant.js:19:11)
    at validate (/home/langpavel/Projects/work/slackday/breedly/node_modules/graphql/validation/validate.js:58:27)
    at /home/langpavel/Projects/work/slackday/breedly/node_modules/express-graphql/dist/index.js:143:52
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by calebmer** [comment]
Could I see some more context? This could be caused by a number of things.

**Comment by langpavel** [comment]
@calebmer Yes, of course, I should describe more.. I was in hurry, sorry.

This problem occurs when you have graphql as direct dependency in your project and you using postgraphql as express middleware. One solution to this is extend supported graphql versions then npm will install only one version of graphql in dependency graph.

**Comment by calebmer** [comment]
Ok makes sense. I wonder if there's an even better solution? Maybe listing `graphql` as a `peerDependency`? That may make things tricky for global installs though.

The solution may be to just allow any version that satisfies the same major version as PostGraphQL expects.

I'll do some research into this, but in the meantime is matching your version with the PostGraphQL version enough?

**Comment by calebmer** [comment]
Extended support as of https://github.com/calebmer/postgraphql/commit/889f3c7bfcb4aba7db86a7e7b22b22287c7806f1 and released in `1.6.1`, tell me if that doesn’t work :+1:
